VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Computer PDU_HIL VCU_HIL


BO_ 2214662671 Batt_thermistor: 3 Computer
 SG_ Batt_thermistor : 0|18@1+ (1,0) [60|200060] "Ohm"  PDU_HIL

BO_ 2282098449 Message_status: 1 VCU_HIL
 SG_ Throttle_B_status : 4|1@1+ (1,0) [0|1] ""  Computer
 SG_ Throttle_A_status : 3|1@1+ (1,0) [0|1] ""  Computer
 SG_ Steering_status : 2|1@1+ (1,0) [0|1] ""  Computer
 SG_ Brake_pos_status : 1|1@1+ (1,0) [0|1] ""  Computer
 SG_ Brake_Pres_status : 0|1@1+ (1,0) [0|1] ""  Computer

BO_ 2214924559 Steering_raw: 2 Computer
 SG_ Steering_raw : 0|12@1+ (1,0) [0|3300] "mV"  VCU_HIL

BO_ 2214859023 Brake_position: 2 Computer
 SG_ Brake_position : 0|12@1+ (1,0) [0|3300] "mV"  VCU_HIL

BO_ 2214793487 Brake_pres_raw: 2 Computer
 SG_ Brake_Pres_Raw : 0|12@1+ (1,0) [0|3300] "mV"  VCU_HIL

BO_ 2214727951 Throttle_position_B: 2 Computer
 SG_ Throttle_position_B : 0|12@1+ (1,0) [0|3300] "mV"  VCU_HIL

BO_ 2214662415 Throttle_position_A: 2 Computer
 SG_ Throttle_position_A : 0|12@1+ (1,0) [0|3300] "mV"  VCU_HIL



CM_ SG_ 2214662415 Throttle_position_A "
";
BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "BusType" "CAN";

